### DISCUSSION TITLE
Open auto-info pop-up from the bottom of the editor

### DISCUSSION THREAD

**Discussion by petejodo** [helix-editor/helix#6442] [Ideas] [answered]
Whenever the auto-info box pops up, it pops up over content that is above the current cursor which a lot of the time blocks useful context e.g. variable names, function, etc.

![Screenshot from 2023-03-18 17-35-26](https://user-images.githubusercontent.com/1938892/227738354-891ce63d-2c93-48ae-901a-789000321588.png)

I've disabled it because it's quite distracting. I'd like to have it pop up from the bottom instead personally. I believe there's something that actually does pop up from the bottom as prior art but it's escaping me right now what it is.

I recognize this may just be a me problem though

**Comment by petejodo** [comment]
my suggestion to open it from the bottom is only because the auto-info can't pop up from underneath the cursor because otherwise it'd overlap with the auto-complete pop up. There may be other ideas that work here like being able to open/close the auto-info box while in insert mode. Actually, I just realized I haven't checked if that's possible in the existing keymap, checking...

**Reply by kirawi** [reply]
You can bind a key to `signature-help` after disabling it automatically opening in your `config.toml`.

**Comment by petejodo** [comment]
looks like there isn't a way to open/close the info pop up while in insert mode, that could possibly be another solution to my issue here

**Reply by kirawi** [reply]
I think `ctrl-c` works.

**Reply by petejodo** [reply]
oh wow it does, thanks so much!

**Comment by petejodo** [comment, marked as answer]
while not technically answered, both responses from Kirawi solve the problem for me so marking this as answered and closing. Since I can't mark sub-comments as answers, copying them here

> `ctrl-c` allows you to close the info box in insert mode

> can bind `signature-help` to keys after disabling the automatic usage of it to open and close it

**Comment by offsetcyan** [comment]
I don't consider this answered either, it would be a helpful feature - but I think Helix lacks the kind of Emacs-ish minibuffer this would want
